File: JavaProjects/FactoryMethod_SimpleFactory/src/boletobancario/BancoCaixaBoleto30Dias.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package boletobancario;

/**
 *
 * @author jeffe
 */
public class BancoCaixaBoleto30Dias extends Boleto {

    public BancoCaixaBoleto30Dias(double valor) {
        this.valor=valor;
        this.juros = 0.05;
        this.desconto=0.05;
        this.multa=0.1;
    }
    
}

File: CsharpProjects/FactoryMethodSolucao/FactoryMethodSolucao/boletobancario/Caixa/BancoCaixaBoleto30Dias.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using FactoryMethodSolucao.boletobancario;

namespace FactoryMethodSolucao.boletobancario.Caixa
{
    public class BancoCaixaBoleto30Dias : Boleto
    {
        public BancoCaixaBoleto30Dias(double valor)
        {
            this.valor = valor;
            juros = 0.05;
            desconto = 0.05;
            multa = 0.1;
        }
    }
}
